### PR TITLE
Document Mazda button entities

### PR DESCRIPTION
**Pull request by bdr99**
## Proposed change

Add documentation for Mazda button entities.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/67597
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/mazda.markdown [hunk 1]
```diff
@@ -38,6 +40,17 @@ This integration requires an active Mazda Connected Services subscription and a
 
 ## Platforms
 
+### Button
+
+Button entities are available to perform the following actions:
+- Start engine (engine can only be started 2 consecutive times; to reset this counter, the vehicle must be driven)
+- Stop engine (only works if the engine was started remotely)
+- Turn on hazard lights (lights will flash briefly and then then turn off)
```
Comment (bdraco) [inline source/_integrations/mazda.markdown]:
```suggestion
- Turn on hazard lights (lights will flash briefly and then turn off)
```

File: source/_integrations/mazda.markdown [hunk 2]
```diff
@@ -38,6 +40,17 @@ This integration requires an active Mazda Connected Services subscription and a
...
+- Turn off hazard lights (if they have been manually turned on from inside the vehicle - lights will turn back on if a door is opened)
+
+Additionally, the following button entity is available for electric vehicles:
+- Refresh vehicle status (only refreshes electric vehicle related information)
```
Comment (bdraco) [inline source/_integrations/mazda.markdown]:
```suggestion
- Refresh vehicle status (only refreshes electric vehicle-related information)
```

**Comment by bdr99** [comment]
@bdraco Addressed your comments.

File: source/_integrations/mazda.markdown [hunk 3]
```diff
@@ -38,6 +40,17 @@ This integration requires an active Mazda Connected Services subscription and a
 
 ## Platforms
 
+### Button
+
+Button entities are available to perform the following actions:
```
Comment (klaasnicolaas) [inline source/_integrations/mazda.markdown:45]:
```suggestion
Button entities are available to perform the following actions:

```

File: source/_integrations/mazda.markdown [hunk 4]
```diff
@@ -38,6 +40,17 @@ This integration requires an active Mazda Connected Services subscription and a
...
+- Start engine (engine can only be started 2 consecutive times; to reset this counter, the vehicle must be driven)
+- Stop engine (only works if the engine was started remotely)
+- Turn on hazard lights (lights will flash briefly and then turn off)
+- Turn off hazard lights (if they have been manually turned on from inside the vehicle - lights will turn back on if a door is opened)
+
+Additionally, the following button entity is available for electric vehicles:
```
Comment (klaasnicolaas) [inline source/_integrations/mazda.markdown:52]:
```suggestion
Additionally, the following button entity is available for electric vehicles:

```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — APPROVED** [review]
✅ LGTM

**Review by bdraco — APPROVED** [review]

**Pull request merged**